src/cobalt_model/permalink.rs

```diff
@@ -0,0 +1,55 @@
+use liquid;
+use std::path::{Path, PathBuf};
+
+use error::*;
+
+pub fn explode_permalink<S: AsRef<str>>(
```

Comment:
Eventually we can consider refactoring this into a newtype so we can directly reference it in the frontmatter, etc and have everything work out.

Reply:
You mean, a proper `struct Permalink`?

Reply:
Yup.  It'll take some iterating on to figure out a design but I don't think its something you should worry about for this.